web/web.go

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 */
 
 import (
+	"flag"
```

Comment:
flag handling goes in cmd/server/server.go, not here.

Reply:
moving it to server

Reply:
moved via 749a9aaa2965121b360899a3085d200192622065